src/examples/buffered_example1.py

```diff
@@ -67,8 +67,7 @@
 # ramp methods or add_mapping_to_instrument(instrument, path) for instruments without
 # to map the instrument's parameters to QuMADA-specific names.
 
-dac = Decadac("dac", "ASRL6::INSTR", min_val=-10, max_val=10, terminator="
```

Comment:
Examplary issue with maybe wrong encoding -> unterminated string literal. Fixed in PR #11.

Reply:
The change was made in commit 2edd8db. That commit added license information to all python files. Somehow escaped characters were replaced (e.g. \t to whitespace as well).

Reply:
Thank you for this investigation!

Reply:
> The change was made in commit 2edd8db. That commit added license information to all python files. Somehow escaped characters were replaced (e.g. \t to whitespace as well).

Sounds to me like a new issue. Should we solve this outside of this PR?

Reply:
I created a separate issue #23.